src/Components/PaperEditor/TranscriptsContainer/index.js

```diff
@@ -31,6 +31,7 @@ const TranscriptsContainer = ({ transcripts, projectId, firebase }) => {
       title={ transcript.title }
       projectId={ projectId }
       firebase={ firebase }
+      trackEvent={ trackEvent }
```

Comment:
Must pass trackEvent to child components, that don't have direct access to analytics context.
The parent components access the context via:
`export default withAuthorization(condition)(ComponentName);`